datalake-dr-connector/src/main/java/com/sequenceiq/cloudbreak/datalakedr/model/DatalakeBackupStatusResponse.java

```diff
@@ -1,40 +1,27 @@
 package com.sequenceiq.cloudbreak.datalakedr.model;
 
+import java.util.List;
 import java.util.Optional;
+import java.util.function.Predicate;
 
-public class DatalakeBackupStatusResponse {
-
-    public static final String NO_FAILURES = "No failure messages found";
-
-    public enum State {
-        STARTED,
-        IN_PROGRESS,
-        SUCCESSFUL,
-        VALIDATION_SUCCESSFUL,
-        VALIDATION_FAILED,
-        FAILED,
-        CANCELLED
-    }
+public class DatalakeBackupStatusResponse implements DatalakeOperationStatus{
```

Comment:
The build will failed.  (missing space before the `{`)
I suggest that run the `gradle build` and fix all of this `PMD` issues. and push your changes.